src/main/java/teammates/common/datatransfer/questions/FeedbackRankRecipientsResponseDetails.java

```diff
@@ -35,49 +35,18 @@ public static List<FeedbackResponseAttributes.UpdateOptions> getUpdateOptionsFor
 
         FeedbackResponseDetails details;
         FeedbackRankRecipientsResponseDetails responseDetails;
-        boolean[] isRankUsed;
+        // boolean[] isRankUsed;
         Set<FeedbackResponseAttributes> updatedResponses = new HashSet<>();
-        boolean isUpdateNeeded = false;
         int answer;
```

Comment:
For the same reason as `details` and `responseDetails` mentioned above, can `answer` be declared in line 56?